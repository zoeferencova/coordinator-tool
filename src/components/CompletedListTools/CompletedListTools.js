import React from 'react';
import './CompletedListTools.css'

export default class CompletedListTools extends React.Component {
    render() {
        return (
            <div className="tools">
                <div>
                    <label htmlFor="search">Search: </label>
                    <input type="text" id="search"></input>
                </div>
                <div>
                    <label htmlFor="sort">Sort by: </label>
                    <select name="sort" id="sort">
                        <option value="none"></option>
                        <option value="advisor">Advisor (A-Z)</option>
                        <option value="project">Project Name</option>
                        <option value="pm">PM</option>
                        <option value="date-desc">Date (Newest-Oldest)</option>
                        <option value="date-asc">Date (Oldest-Newest)</option>
                        <option value="status">Status</option>
                    </select>
                </div>
            </div>
        )
    }
}