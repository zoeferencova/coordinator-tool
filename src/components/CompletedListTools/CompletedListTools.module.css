.tools {
    display: flex;
    flex-direction: row;
    justify-content: flex-start;
}

.hide {
    display: none;
}
