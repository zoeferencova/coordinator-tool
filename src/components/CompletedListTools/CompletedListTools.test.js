import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import CompletedListTools from './CompletedListTools'

describe(`Completed List Tools component`, () => {
  it('renders a .CompletedListTools by default', () => {
    const wrapper = shallow(<CompletedListTools />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})