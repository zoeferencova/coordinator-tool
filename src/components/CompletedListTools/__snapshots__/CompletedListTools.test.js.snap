// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Completed List Tools component renders a .CompletedListTools by default 1`] = `
<div
  className="tools"
>
  <div>
    <label
      htmlFor="search"
    >
      Search: 
    </label>
    <Input
      id="search"
      onChange={[Function]}
      type="text"
    />
  </div>
</div>
`;
